doc/content/microreactors/s8er/s8er_multiphysics_model.md

```diff
@@ -0,0 +1,245 @@
+| 700      |   700   |  7    | Fuel Channel |
+| 800      |   800   |  8    | Fuel Channel |
+| 300      |   300   |  9, 10 | Coolant Channel |
+| 1200     |   1200  |  11   | Internal Reflector + Core Barrel |
+
+!media s8er/griffin_universe_map.png
+  caption= S8ER Griffin Mesh, from [!citep](s8er_naupa2022)
+  style=width:100%;margin-left:auto;margin-right:auto
+
+### TransportSystems
+
+As stated previously, the Griffin model uses the newly optimized DFEM-SN transport scheme to solve for the eigenvalue. Here a 16 group energy structure is used with vacuum boundary conditions on the outeredge of the core barrel. The solver uses 9 azimuthal and 3 polar angles, typically the default for these are 6 and 2 respectively but because this highly anistropic, a higher order of angles is required for solution fidelity. The benefit with using the DFEM-SN scheme rather than the stand CFEM-Diffusion scheme, is the removal of several layers of calculation for the same fidelity, however at the expense of a more expensive computation. The DFEM-SN scheme is optimized and scales well with problem size. 
+
+!listing microreactors/s8er/core_2D_griffin_coupled.i block=TransportSystems
+
+### Materials
+
+Through the use of the [CoupledFeedbackMatIDNeutronicsMaterial](https://griffin-docs.hpcondemand.inl.gov/latest/source/materials/CoupledFeedbackMatIDNeutronicsMaterial.html#coupledfeedbackmatidneutronicsmaterial), by the mapping of `materialIDs` to the mesh done earlier, this object conviniently handles the direct mapping of neutronic data into the mesh. The object also has a feature to correct the neutronic data used in a material by adjusting the cross sections for a correction volume. This is done for the internal reflector and core barrle region to more appropiately match the reference model.
+
+!listing microreactors/s8er/core_2D_griffin_coupled.i block=Materials
+
+## Bison Model
+
+Because bison is only taking the power distribution from the griffin model, we have the freedom to model each fuel channel component explciitly and hence we are able to capture unique heat transfer phenomena such as the gap radiative transfer and model thermal properties for each component. This model only captures heat conduction but future updates will include feedback from thermomechanical effects such as thermal expansion. 
+
+!media s8er/fuelpin_components_label_bison.png
+  caption= S8ER Bison Mesh, from [!citep](s8er_naupa2022)
+  style=width:100%;margin-left:auto;margin-right:auto
+
+### Geometry 
+
+The process for generating the Bison mesh is almost identical to the process described for the Griffin mesh, with a few extra steps. This time since were modeling the explicit fuel element geometry, we use the [PolygonConcentricCircleMeshGenerator](https://mooseframework.inl.gov/source/meshgenerators/PolygonConcentricCircleMeshGenerator.html) to model concentric rings correpsondig to the outer radii of each material in the fuel element.
```

Comment:
```suggestion
The process for generating the Bison mesh is almost identical to the process described for the Griffin mesh, with a few extra steps. This time since we are modeling the explicit fuel element geometry, we use the [PolygonConcentricCircleMeshGenerator](https://mooseframework.inl.gov/source/meshgenerators/PolygonConcentricCircleMeshGenerator.html) to model concentric rings corresponding to the outer radii of each material in the fuel element.
```